flask/app.py

```diff
@@ -157,6 +157,11 @@ class Flask(_PackageBoundObject):
     #: :class:`~flask.Response` for more information.
     response_class = Response
 
+    #: The class that is used for the Jinja environment.
+    #:
+    #: .. versionadded:: 0.11
```

Comment:
I always assumed it is 1.0 but I'm not sure whether we should release an intermediate version.